### ISSUE TITLE
Mallika's codepath app

### ISSUE THREAD

**Issue by mallikarao** [mallikarao/codepath_myapp#1]
Please review my prework. /cc @codepathreview @codepath

I am brand new to android development. Could not fully complete the Intent, edit text part. It's in bits.
Still submitting.
Trying till the 22th, nevertheless.

Hope to make it there.
Mallika

**Comment by nesquena** [comment]
Hi Mallika,

Thanks for letting us know and glad to see you got through the first few stories. Feel free to ask if you have specific questions and please ping us back here once you've completed the editing and other missing elements. I'm sure you can complete it and then we can setup a time to chat with you.

**Comment by mallikarao** [comment]
Thanks  /cc @codepathreview @codepath, am trying out the editting part.
I actually have the activity_edit_item.xml with the button named as "save" and EditText with input type multi line text etc.

I tried to put a section in MainActivity which can do something when an item is clicked.
lvItems.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

```
        }
    });
```

I think here is where I'll have to put the Intent stuff. Do you think I am in the right direction ?

**Comment by nesquena** [comment]
Yes you are definitely on the right track. Intents move the app to a new screen. When the user clicks, you want to use the intent to bring them to the edit screen.

**Comment by mallikarao** [comment]
Thanks /cc @codepathreview @codepath.
Could not get the intents fully up and running. I was hoping I will last week. But there is more learning in it for me before I get things working.
Hope I still get selected for the course, I can learn.
Thanks for the pointers on the way, they helped.

**Comment by nesquena** [comment]
Unfortunately the class is already full at this point, but we are happy to add you to the waitlist. For those not admitted as participants, we can also offer access to applicants as a remote observer cohort that has access to the materials and is able to work on the projects at their own pace. I'd encourage you to continue working on the pre-work as a learning exercise. Let me know if you'd like to follow along as a remote observer and I'll send you more details soon.
